snippets/provider_listenable.code-snippets

```diff
@@ -0,0 +1,11 @@
+{
+  "Provider Listenable": {
+    "scope": "dart",
+    "prefix": "listen",
+    "description": "Create a provider listenable",
+    "body": [
+      "ref.listen<$1>($2, (value) { ", 
+      "\t}", 
+      ");"]
```

Comment:
This should be something like this (of course using the quotation marks as well)

```dart
ref.listen<$1>($2, (value) {  
"\t$3", 
});
```